pino.d.ts

```diff
@@ -791,15 +791,15 @@ declare namespace pino {
  * relative protocol is enabled. Default: process.stdout
  * @returns a new logger instance.
  */
-declare function pino<Options extends LoggerOptions | DestinationStream>(optionsOrStream?: Options): Logger<Options>;
+declare function pino<CustomLevels extends string = never>(optionsOrStream?: LoggerOptions<CustomLevels> | DestinationStream): Logger<CustomLevels>;
 
 /**
  * @param [options]: an options object
  * @param [stream]: a writable stream where the logs will be written. It can also receive some log-line metadata, if the
  * relative protocol is enabled. Default: process.stdout
  * @returns a new logger instance.
  */
-declare function pino<Options extends LoggerOptions>(options: Options, stream: DestinationStream): Logger<Options>;
+declare function pino<CustomLevels extends string>(options: LoggerOptions<CustomLevels>, stream: DestinationStream): Logger<CustomLevels>;
```

Comment:
Shouldn't it be:
<pre>
declare function pino&lt;CustomLevels extends string <b>= never</b>&gt;(options: LoggerOptions<CustomLevels>, stream: DestinationStream): Logger<CustomLevels>;
</pre>
 
or am I missing something? @mcollina @UndefinedBehaviour

Reply:
@dpetzev fixed here https://github.com/pinojs/pino/pull/1871

Reply:
Appreciate it!